src/inmanta/logging.py

```diff
@@ -834,9 +837,13 @@ def _get_logger_name_for(self, logger_name: str) -> str:
             if match:
                 # Log record created by an Inmanta module.
                 return match.groupdict()["module_name"]
+            elif "export" in logger_name:
+                return "exporter"
+            elif "protocol" in logger_name:
+                return "exporter"
```

Comment:
I don't get this one.

Reply:
We try to shorten logger names when running the compiler to either 'compiler' or 'exporter'

This shorten all log lines in protocol to 'exporter', because only the exporter uses the protocol layer